core/domain/exp_services.py

```diff
@@ -1382,6 +1383,26 @@ def get_average_rating_from_exp_summary(exp_summary):
     return 0
 
 
+def get_lower_bound_wilson_score_from_exp_summary(exp_summary):
```

Comment:
get_scaled_average_rating_from_exp_summary seems like a better name to me.

Note that the 1 + 4 * result and the (result - 1)/4 represent a linear scaling between the intervals [1, 5] (the original star rating value) and [0, 1]. The thing that's being returned at the end is a pseudo-rating -- the aim of the last conversion is to convert the value in [0, 1] to a "number of stars".